src/bin.ts

```diff
@@ -95,6 +96,8 @@ const createSafeS3Key = (key: string): string => {
 
 const deploy = async ({ yes, bucket }: { yes: boolean, bucket: string }) => {
     const spinner = ora({ text: 'Retrieving bucket info...', color: 'magenta' }).start();
+    
+    const uploadQueue: any[] = [];
```

Comment:
I think this should be typed as `Array<() => Promise<void>>` or `(() => Promise<void>)[]`.